pages/productPage/productPage.css

```diff
@@ -0,0 +1,32 @@
+.grid-container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    "nav-bar nav-bar nav-bar nav-bar"
+    "filter product product product";
+}
```

Comment:
nit on naming

filter -> sidebar
product -> content


as tomorrow you can change the filter to something else and product can be services as well.

Reply:
Resolved.